src/main/java/hudson/plugins/promoted_builds/parameters/PromotedBuildParameterDefinition.java

```diff
@@ -113,10 +138,24 @@ public String getProcess() {
         return promotionProcessName;
     }
 
+    @Nonnull
+    @Deprecated
     public List getBuilds() {
```

Comment:
This call is not very reliable, because the relative addressing works for the UI only. I'll add the Javadoc